src/renderers/index.css

```diff
@@ -0,0 +1,287 @@
+/*-----------------------------------------------------------------------------
+| Copyright (c) Jupyter Development Team.
+| Distributed under the terms of the Modified BSD License.
+|----------------------------------------------------------------------------*/
+
+
+.jp-RenderedHTMLCommon {
+
+  color: black;
+  font-size: 14px;
+  line-height: 20px;
+
+}
+
+.jp-RenderedHTMLCommon .MathJax_Display {
+  margin: 0;
+}
+
+
+.jp-RenderedHTMLCommon em {
+  font-style: italic;
+}
+
+.jp-RenderedHTMLCommon strong {
+  font-weight: bold;
+}
+
+.jp-RenderedHTMLCommon u {
+  text-decoration: underline;
+}
+
+.jp-RenderedHTMLCommon:link {
+  text-decoration: underline;
+}
+
+.jp-RenderedHTMLCommon:visited {
+  text-decoration: underline;
+}
+
+/*For a 14px base font size this goes as:
+font-size = 26, 22, 18, 14, 12, 12
+margin-top = 14, 14, 14, 14, 8, 8
+*/
+
+.jp-RenderedHTMLCommon h1 {
+  font-size: 185.7%;
+  margin: 1.08em 0 0 0;
+  font-weight: bold;
+  line-height: 1.0;
+}
+
+
+.jp-RenderedHTMLCommon h2 {
+  font-size: 157.1%;
```

Comment:
Subtle reasons related to establishing a fixed baseline in the typography. Also, following the classic notebook on these.